spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -265,19 +265,19 @@
     end
 
     describe "wages" do
-      context "when no w2 wages (line 15 is -1)" do
-        let(:intake) { create(:state_file_nj_intake, :df_data_minimal) }
+      let(:intake) { create(:state_file_nj_intake) }
 
+      context "when no w2 wages (line 15 is -1)" do
         it "does not include WagesSalariesTips item" do
+          allow_any_instance_of(Efile::Nj::Nj1040Calculator).to receive(:calculate_line_15).and_return -1
```

Comment:
[DUST] Seeing a change in testing strategy here. Previously we would create the intake and then let the calculator run before checking the XML, now we're mocking out the calculator values and validating that the XML is generated properly. Personally, I'm fine with this, since we test the calculator separately in its own spec, we don't need to have "end-to-end" testing here.

@mmazanec22 @aloverso worth looking at!

Reply:
Got it! I agree that mocking out the calculator is the right way to go for the XML builder spec. This is something that we've discussed internally as a team, but never took the time to refactor previous tests.

I have no objections here, just flagging for the team to look over.